core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -452,6 +458,9 @@
       "The time to wait for a tablet server to process a bulk import request."),
   TSERV_MINTHREADS("tserver.server.threads.minimum", "20", PropertyType.COUNT,
       "The minimum number of threads to use to handle incoming requests."),
+  TSERV_MINTHREADS_ALLOW_TIMEOUT("tserver.server.thread.timeout.allowed", "true",
```

Comment:
The default of true maintains the current behavior, however that behavior seemed problematic in the issue you opened.  Makes me wonder if the default should be false.

Reply:
I thought about that.  I think this only really becomes a problem when you are dealing with very large numbers of threads (300+ maybe) which is why I left the default as true.  I am certainly will to change it to false however.

Reply:
I am slightly in favor of false because it addresses a known problem and seems like it should not cause problems, but it could.  The way this change is structured it does not change all of the existing code to false just a few select places, which is good.  Leaving most of the internal usage w/ a hard coded true is good and changing just these configurable ones to false may be a good way to go.

Reply:
ok, I am game for that.  I will change it now.